pavelib/assets.py

```diff
@@ -733,20 +742,16 @@ def watch_assets(options):
     if tasks.environment.dry_run:
         return
 
-    themes = getattr(options, 'themes', None)
-    theme_dirs = getattr(options, 'theme-dirs', [])
-
+    themes = get_parsed_option(options, 'themes')
+    # option with hyphen(-) is converted to underscore(_) by `cmdopts` so `theme_dirs` will be passed
+    # instead `theme-dirs`. Behavior mentioned in `cmdopts()` of python2.7/site-packages/paver/tasks.py.
+    theme_dirs = get_parsed_option(options, 'theme_dirs', [])
     if not theme_dirs and themes:
         # We can not add theme sass watchers without knowing the directory that contains the themes.
         raise ValueError('theme-dirs must be provided for watching theme sass.')
     else:
         theme_dirs = [path(_dir) for _dir in theme_dirs]
```

Comment:
@noraiz-anwar, what are our expectations in `theme_dirs` at L753 in both cases before and after your fix? I think `theme_dirs` could also be of type string before your fix and after the fix, its always list?